harmonica/tests/test_point_gravity.py

```diff
@@ -89,237 +90,105 @@ def test_invalid_masses_array():
 # ---------------------------
 # Cartesian coordinates tests
 # ---------------------------
-@pytest.mark.use_numba
-def test_potential_cartesian_known_values():
+
+
+@pytest.fixture(name="point_mass")
+def fixture_point_mass():
```

Comment:
Since there are a lot of small changes, I'm highlighting the most interesting pieces of diffs.

Here I created some fixtures for the precomputed results.